benchmark/Streamly/Benchmark/Data/Fold.hs

```diff
@@ -307,14 +307,6 @@ o_1_space_serial_elimination value =
               value
               "mean"
               (S.fold FL.mean . S.map (fromIntegral :: Int -> Double))
-        , benchIOSink
-              value
-              "variance"
-              (S.fold FL.variance . S.map (fromIntegral :: Int -> Double))
-        , benchIOSink
-              value
-              "stdDev"
-              (S.fold FL.stdDev . S.map (fromIntegral :: Int -> Double))
```

Comment:
We should not remove the benchmarks yet. We can remove those when we remove these functions altogether. Same about tests.

Can we suppress the warnings?

Reply:
We can.